.github/workflows/tests.yml

```diff
@@ -38,31 +37,20 @@ jobs:
           # 2025
           - python: "3.14"
             geos: 3.14.1
-            numpy: 2.3.3
+            numpy: 2.3.5
           # free threaded Python (no numpy version to indicate installing nightly cython and numpy)
-          - os: ubuntu-22.04
+          - os: ubuntu-latest
```

Comment:
I think we used a different os entry here than the default in the matrix, because otherwise that overrides the other one? (for the pypy entry below there is an explicit comment to that effect)